### PR TITLE
fix: landing page search 

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
https://github.com/ansys/ansys-sphinx-theme/issues/571#issuecomment-2443923015

### CODE REVIEW

**Review by jorgepiloto — APPROVED** [review]

**Review by RobPasMue — APPROVED** [review]

**Review by germa89 — APPROVED** [review]

File: doc-deploy-dev/action.yml [hunk 1]
```diff
@@ -315,6 +317,7 @@ runs:
           sed -i 's/action="search.html"/action="version\/dev\/search.html"/g' index.html
           sed -i 's|script\.src = ".*_static/js/meilisearch_theme_wrap\.js"|script.src = "version/dev/_static/js/meilisearch_theme_wrap.js"|g' index.html
           sed -i 's|const SEARCH_FILE = ".*_static/search.json";|const SEARCH_FILE = "version/dev/_static/search.json";|g' index.html
+          sed -i 's|var searchPath = ".*_static/search.json";|var searchPath = "version/dev/_static/search.json";|g' index.html
```
Comment (germa89) [inline doc-deploy-dev/action.yml:320]: I hope you will remember to delete this one too :P

**Review by germa89 — APPROVED** [review]

File: doc-deploy-dev/action.yml (hunk 1 above)
Comment (germa89) [reply doc-deploy-dev/action.yml:320]: Looooool

Comment (Revathyvenugopal162) [reply doc-deploy-dev/action.yml:320]: > I hope you will remember to delete this one too :P

yes, i will open an issue for this for us to remember

**Pull request merged**